vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -37,18 +44,29 @@ public class ExhibitionConfigurationController {
 
     @Autowired
     private ExhibitionFactory exhibitFactory;
+    
+    @Autowired
+    private ExhibitionLanguageConfig exhibitionLanguageConfig;
+    
+
 
     @RequestMapping("/staff/exhibit/config")
     public String showExhibitions(Model model) {
         // for now we assume there is just one exhibition
         IExhibition exhibition = exhibitManager.getStartExhibition();
         if(exhibition!=null) {
             model.addAttribute("exhibition", exhibition);
+            if(exhibition.getLanguages() != null ) {
+                model.addAttribute("mappedLanguages", exhibition.getLanguages()
+                        .stream().map(language -> language.getLabel()).collect(Collectors.toList()));
```

Comment:
what are "mappedLanguages"? and why not putting the language list in the model (not just the labels), you probably need them somewhere (or if not, it's just a `.label` more to show the language names).

Reply:
mappedLanguages is just used to highlight the already mapped languages to an exhibition, in the selection dropdown. This is to avoid adding any more logic in front end. 
The entire language object is being sent  to populate the dropdown in "languageList".

Reply:
ah, makes sense. It needs a better name though. It's not clear what "mappedLanguages" would be in this context.